RecoTauTag/RecoTau/plugins/TauDiscriminationAgainstElectronMVA6.cc

```diff
@@ -132,23 +149,29 @@ TauDiscriminationAgainstElectronMVA6<TauType, TauDiscriminator, ElectronType>::d
           }
           // veto taus that go to ECal crack
 
-          if (std::abs(tauEtaAtECalEntrance.first) < ECalBarrelEndcapEtaBorder) {  // Barrel
+          if (std::abs(tauEtaAtECalEntrance.first) < ecalBarrelEndcapEtaBorder_) {  // Barrel
             if (numSignalGammaCandsInSigCone == 0 && hasGsfTrack) {
               category = 5.;
             } else if (numSignalGammaCandsInSigCone >= 1 && hasGsfTrack) {
               category = 7.;
             }
-          } else {  // Endcap
+          } else if (!isPhase2_ || (std::abs(tauEtaAtECalEntrance.first) < ecalEndcapVFEndcapEtaBorder_)) {  // Endcap
```

Comment:
```suggestion
          } else if (!isPhase2_ || std::abs(tauEtaAtECalEntrance.first) < ecalEndcapVFEndcapEtaBorder_) {  // Endcap
```
doesn't follow the style of other conditionals (and superfluous)

Reply:
OK

Reply:
`code-format` is fine with removal of parentheses around the second condition (which are indeed not necessary), but not with line break after `||`.

Reply:
I did not add a line break in the suggestion, it's a matter of rendering of the proposed diff on the web

Reply:
Thanks for clarifications; parentheses are anyway removed now.